Configuration/Eras/python/Modifier_run2_tauIDformat_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+run2_tauIDformat =  cms.Modifier()
```

Comment:
why does this have run2_ prefix? IIRC, #28417 added tau container-based ID for all data/simulation eras/ranges.

I was actually expecting that the modifier will add the sequence/task to rerun tau IDs rather than insert support to read the old IDs, but for this case it seems like your solution is less invasive.

I propose to change the name of the modifier to ```tau_readOldDiscriminator ``` or something similar

Reply:
My reasoning was that files produced during run2 contain the old format (which could be called run2_tauIDformat), but I'm fine with calling it `tau_readOldDiscriminator` which better reflects what it is doing. However, one might think that this means to switch to a different ID rather than just an old format. Would `tau_readOldDiscriminatorFormat` also suitable or is it too long?

Reply:
"during run2" is just a calendar period, there are AOD files produced for Run-1, Run-3 and Phase-2 during that period. It would be rather confusing to use run2_ prefix for a modifier for these cases as well if it becomes necessary.

Reply:
true